es2csv.py

```diff
@@ -99,6 +99,9 @@ def next_scroll(scroll_id):
             size=self.scroll_size,
             terminate_after=self.opts.max_results
         )
+        if self.opts.doc_type:
+            search_args['doc_type']=self.opts.doc_type
```

Comment:
Please format this code according to [PEP 8](https://www.python.org/dev/peps/pep-0008/).